dataplicity/m2m/commandservice.py

```diff
@@ -38,54 +38,83 @@ def __init__(self, channel, command):
     def __repr__(self):
         return self._repr
 
+    @classmethod
+    def send_error(cls, channel, status, msg, **extra):
+        """Send a control packet with an error"""
+        error = {
```

Comment:
This forms a sub-protocol of sorts. And will be a standard way of reporting service related errors.